unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/UIAbstractElements/UIElementHandlerBase.cs

```diff
@@ -0,0 +1,59 @@
+﻿using DCL.Controllers;
+using DCL.ECS7.InternalComponents;
+using DCL.Models;
+using UnityEngine.UIElements;
+
+namespace DCL.ECSComponents.UIAbstractElements
+{
+    /// <summary>
+    /// Base class to handle common routines such as
+    /// adding and removing UI Components and Visual Elements
+    /// </summary>
+    public abstract class UIElementHandlerBase
```

Comment:
all this logic could be replaced by extending `IInternalECSComponent<InternalUiContainer>` like we have for other components, for example https://github.com/decentraland/unity-renderer/blob/dev/unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/Interfaces/Extensions/InternalCollidersExtensions.cs
instead of adding this abstract class